justfile

```diff
@@ -10,6 +10,9 @@ export RUSTFLAGS := env('RUSTFLAGS', if ci_mode == '1' {'-D warnings'} else {''}
 export RUSTDOCFLAGS := env('RUSTDOCFLAGS', if ci_mode == '1' {'-D warnings'} else {''})
 export RUST_BACKTRACE := env('RUST_BACKTRACE', if ci_mode == '1' {'1'} else {''})
 
+export MLN_REPO := `cargo metadata --format-version 1 --no-deps | jq -e -r '.packages[] | select(.name == "maplibre_native") | .metadata.mln.repo // error("MLN repo missing from Cargo metadata")'`
```

Comment:
you can use  `just get-crate-field 'metadata.mln.repo'` to get it - same as what `get-msrv` does